src/components/RegistrationList.tsx

```diff
@@ -129,21 +136,21 @@ export const RegistrationListItemContent = ({
             marginBottom: '0.5rem',
             flexWrap: 'wrap',
           }}>
-          <RegistrationIconHeader
-            publicationInstanceType={registration.type}
-            publicationDate={registration.publicationDate}
-          />
-          {ticketView &&
-            (registration.recordMetadata.status === RegistrationStatus.Draft ||
-              registration.recordMetadata.status === RegistrationStatus.New) && (
-              <Typography
-                sx={{
-                  p: '0.1rem 0.75rem',
-                  bgcolor: 'warning.light',
-                }}>
-                {t('registration.public_page.result_not_published')}
-              </Typography>
-            )}
+          {!ticketView && (
+            <RegistrationIconHeader
+              publicationInstanceType={registration.type}
+              publicationDate={registration.publicationDate}
+            />
+          )}
+          {ticketView && (
+            <HorizontalBox sx={{ gap: '1rem', marginBottom: '0.5rem', flexWrap: 'wrap' }}>
+              {ticketType && <TicketTypeTag type={ticketType} />}
+              <PublicationInstanceText publicationInstanceType={registration.type} />
+              {registration.publicationDate?.year && <DateText publicationDate={registration.publicationDate} />}
+              {(registration.recordMetadata.status === RegistrationStatus.Draft ||
+                registration.recordMetadata.status === RegistrationStatus.New) && <NotPublishedTag />}
+            </HorizontalBox>
+          )}
```

Comment:
This change is because the ticket list item now have two different top lines depending on whether or not its a task or not. New top line on task:

<img width="1089" height="149" alt="image" src="https://github.com/user-attachments/assets/237880cc-61e6-4f15-8f03-f5f39b20f691" />

I made reusable components for what was common, but this whole component should be refactored into two seperate ones eventually